Feedback PR (#1)
ImXron/pytest-stress: README.rst

=============
Pytest-stress
=============

.. image:: https://img.shields.io/pypi/v/pytest-stress.svg
    :target: https://pypi.org/project/pytest-stress
    :alt: PyPI version

.. image:: https://img.shields.io/pypi/pyversions/pytest-stress.svg
    :target: https://pypi.org/project/pytest-stress
    :alt: Python versions

.. image:: https://travis-ci.org/ImXron/pytest-stress.svg?branch=master
    :target: https://travis-ci.org/ImXron/pytest-stress
    :alt: See Build Status on Travis CI

> A plugin that allows you to loop tests for a user defined amount of time.

Reviewer: ```suggestion
A plugin that allows you to loop tests for a user-defined amount of time.
```

Requirements
------------

Only tested with `Pytest`_ version 5.2.2.

Installation
------------

> You can install "pytest-stress" via `Pip`_ from `PyPI`_ (highly recommend installing in a `Pipenv`_)::

Reviewer: ```suggestion
You can install "pytest-stress" via `pip`_ from `PyPI`_ (highly recommend installing in a `Pipenv`_)::
```

$ pip3 install pytest-stress

Usage
-----

Loop tests for 30 seconds::

    $ pytest --seconds 30

Loop tests for 45 minutes::

    $ pytest --minutes 45

Loop tests for 8 hours::

    $ pytest --hours 8

Loop tests for 1 hour 8 minutes and 9 seconds::

    $ pytest --hours 1 --minutes 8 --seconds 9

You can also add these values to config files::

    [pytest]
    addopts = --hours 1 --minutes 30

> Note: These loop times include setup and tear down operations as well. So if you have a test setup that takes 5

Reviewer: ```suggestion
Note: These loop times include setup and teardown operations as well. So if you have a test setup that takes 5
```

Author: Lol my copy and paste fail 🤦‍♂
